manifests/t/tastyworks/tastyworks/1.22.0/tastyworks.tastyworks.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: tastyworks.tastyworks
+PackageVersion: 1.22.0
+PackageLocale: en-US
+Publisher: tastyworks, inc.
+PackageName: tastyworks
+License: © 2017-2021 tastyworks, Inc.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate